### DISCUSSION TITLE
transform_target does not invert transformation for prediction

### DISCUSSION THREAD

**Discussion by thebetauser** [pycaret/pycaret#2724] [Q&A]
Hello,

When setting transform_target=True in the setup() function, the data is transformed correctly however when finalizing the model and calling predict_model, the predicted value which is returned ("Label") is the transformed value instead of being inversed.

Is there a way to have the predict_model function return the inverse value of the label?

**Comment by thebetauser** [comment]
Can anyone confirm if this is a bug or have a workaround?

**Reply by ngupta23** [reply]
> 



> Can anyone confirm if this is a bug or have a workaround?

Will need to look into this and get back to you.

**Reply by ngupta23** [reply]
Please track this issue here: https://github.com/pycaret/pycaret/issues/2771
